src/main/play-doc/developer/DevQuickStart.md

```diff
@@ -16,21 +16,12 @@ The focus of this section is to get started quickly. The full documentation of t
 * [[Deploying bundles|DeployingBundles]]
 * [[ConductR sandbox cluster|ConductrSandbox]]
 
-## Vendor lock-in
```

Comment:
Removed the `Vendor lock-in` chapter. It shouldn't be part of a quick start. In the previous section we are talking that this page focuses to get started quickly. Afterwards the next section `Vendor lock-in` talks about non-quick-start related topics which breaks the reading flow.

@huntc: Should we keep it removed or should we put this information to another chapter?